content/en/docs/instrumentation/js/instrumentation.md

```diff
@@ -3,135 +3,180 @@ title: Instrumentation
+Manual instrumentation is the process of adding observability code to your
+application.
 
-## Example Application
+## Initialize Tracing
 
-In the following this guide will use the following sample app:
+To start [tracing](/docs/concepts/signals/traces/#tracing-in-opentelemetry),
+you'll need to have an initialized
+[`TracerProvider`](/docs/concepts/signals/traces/#tracer-provider) that will let
+you create a [`Tracer`](/docs/concepts/signals/traces/#tracer).
 
-```javascript
-'use strict';
+### Node.js
 
-for (let i = 0; i < 10; i += 1) {
-  doWork();
-}
+To initialize tracing with the Node.js SDK, first ensure you have the SDK
+package and OpenTelemetry API installed:
 
-function doWork() {
-  console.log('work...');
-  // simulate some random work.
-  for (let i = 0; i <= Math.floor(Math.random() * 40000000); i += 1) {}
-}
+```shell
+npm install \
+  @opentelemetry/api \
+  @opentelemetry/sdk-node
 ```
 
-## Initializing a Tracer
+It's recommended to have a separate `tracing.js` file that has all the SDK
+initialization code in it:
+
+```js
+import opentelemetry from "@opentelemetry/api";
+import nodeOTel from "@opentelemetry/sdk-node";
+
+const sdk = new nodeOTel.NodeSDK({
+  traceExporter: new opentelemetry.tracing.ConsoleSpanExporter(),
+});
+
+sdk.start()
+
+// Now you can create a tracer. Export it for the rest
+// of the app to use.
+export const tracer = opentelemetry.trace.getTracer(
+  'name-of-tracer-goes-here'
+);
+```
+
+Next, ensure that `tracing.js` is required in your node invocation. For example:
+
+```
+node --require './tracing.js' <other-app-file.js>
+```
 
-As you have learned in the previous [Getting Started](../getting-started) guide you
-need a TracerProvider and an Exporter. Install the dependencies and add them to
-the head of your application code to get started:
+Although it's not technically required to `--require` the tracing initialization
+for only manual instrumentation, it is necessary when you use automatic
+instrumentation. So in practice, it's still likely to be necessary.
+
+### Browser
+
+First, ensure you've got the right packages:
 
 ```shell
-npm install @opentelemetry/api
-npm install @opentelemetry/sdk-trace-base
+npm install \
+  @opentelemetry/api \
+  @opentelemetry/sdk-trace-web
 ```
 
-Next, initialize a tracer, preferably in a separate file (e.g., `instrumentation-setup.js`):
+Create a `tracing.js` file that initialized the Web SDK, creates a
+`TracerProvider`, and exports a `Tracer`.
 
 ```javascript
-const {
-  BasicTracerProvider,
-  ConsoleSpanExporter,
-  SimpleSpanProcessor,
-} = require('@opentelemetry/sdk-trace-base');
-const opentelemetry = require('@opentelemetry/api');
+import opentelemetry from "@opentelemetry/api";
+import { WebTracerProvider } from '@opentelemetry/sdk-trace-web';
 
-const provider = new BasicTracerProvider();
+const provider = new WebTracerProvider();
 
-// Configure span processor to send spans to the exporter
```

Comment:
(needs to be brought back in likely)

> Batch span processor makes sense in some cases (typical server app) but not in others (serverless). It might make sense to add a comment about this because a serverless container may be killed before a batch is exported.